logdna/resource_view_test.go

```diff
@@ -15,7 +15,7 @@ func TestView_expectInvalidURLError(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config:      testViewInvalidURL(),
-				ExpectError: regexp.MustCompile("Error: Error during HTTP request: Post \"http://api.logdna.co/v1/config/view\": dial tcp: lookup api.logdna.co: no such host"),
+				ExpectError: regexp.MustCompile("Error: Error during HTTP request: Post \"http://api.logdna.co/v1/config/view\": dial tcp: lookup api.logdna.co"),
```

Comment:
Error message differs per OS, so stop checking the message beyond the first part that talks about the bad host.